Add small note about timescale dependent objects (#165)
timescale/docs: timescaledb/tutorials/custom-timescaledb-dashboards.md

    AND ch.chunk_name = cds.chunk_name;
```

> <highlight type="warning">
> Note the view is dependent on TimescaleDB internals and maybe will need to be
> dropped to upgrade the TimescaleDB extension and recreated after the upgrade.
> </highlight>

Reviewer: ```suggestion
<highlight type="warning">
The view is dependent on TimescaleDB internals. You might need to drop the view
to upgrade the TimescaleDB extension, and recreate it after the upgrade.
</highlight>
```
